scripts/data_ingestion/ingest_espn_plays.py

```
@@ -0,0 +1,195 @@
+                    # Game doesn't have play-by-play data available, skip it
+                    print(f"Skipping game {game_id} - no play-by-play data available")
+                    skipped_count += 1
+                else:
+                    print(f"Skipping game {game_id} - empty plays array")
+                    skipped_count += 1
+            
+            print(f"Batch complete: {success_count} success, {failed_count} failed, {skipped_count} skipped")
+        
+        print(f"\nFinal results: {success_count} success, {failed_count} failed, {skipped_count} skipped out of {total_games} games")
+        
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)